kms/src/crypto.rs

```diff
@@ -0,0 +1,25 @@
+use anyhow::{Context, Result};
+use k256::ecdsa::{RecoveryId, Signature, SigningKey};
+use sha3::{Digest, Keccak256};
+
+use ra_tls::kdf;
+
+pub(crate) fn derive_k256_key(
```

Comment:
I think it would be worth commenting on how apps are prevented from accessing each others keys

Reply:
The app-id is simply placed into the derivation path. I've moved the logic to this function for clarity.
https://github.com/Dstack-TEE/dstack/pull/91/commits/01b814e0725e28509a4d9eec621f11f90829a94f